src/battle/overlay_12_0224E4FC.c

```diff
@@ -1714,8 +1714,8 @@ void BattleContext_Init(BATTLECONTEXT *ctx) {
     ctx->unk_2174 = 0;
     ctx->unk_2178 = 0;
     
-    //related to stat (maybe status?) changes
-    ctx->unk_88 = 0;
+    //related to stat changes
+    ctx->statChangeType = 0;
```

Comment:
realistically these might be better off as constants

Reply:
I need to see them used more just to figure out what would map to what but otherwise I agree